### DISCUSSION TITLE
Official Language English

### DISCUSSION THREAD

**Discussion by Malunke** [mikopbx/Core#270] [General ( Обо всём )]
Dear All,

my russian skills are very poor (I had it 4 years in school, but this is quite a long while in the past).

Is it possible to change the official language in github as well as in your forum to english so that many people can participate in your discussions?

I have big problems reading your posts - it is quite impossible for me to understand the simple change-logs of new releases not to mention problems and solutions in the forum posts.

It would be very good when as much as possible could be written in English instead of Russian - even if that is your native language.

Thanks a lot for your help

**Comment by jorikfon** [comment]
Hello. We will try! :)
